groupsforsections.py

```diff
@@ -149,6 +149,33 @@ def create_group(groups_for_section, group_category_id, group_name, course_id):
     return str(group['id'])
 
 
+def add_users_to_group(groups_for_section, group_id, user):
+    """
+    adding user to a group, parses Json, do exception handling
+    :param groups_for_section:
+    :param group_id:
+    :param user:
+    :type groups_for_section GroupsForSections
+    :type group_id str
+    :type user str
+    :return: membership_id
+    :rtype: int, None
+    """
+    logging.debug(add_users_to_group.__name__ + '() called')
+    try:
+        response = groups_for_section.add_users_to_group(group_id, user)
+
+    except requests.exceptions.RequestException as e:
```

Comment:
I'm not sure, but I think Dave was asking what happens if some other kind of exception is raised?  While Requests will raise `RequestException` for the problems it knows about, there could be other exceptions as well.

I that case, the exception will go unhandled by this method and will go back up the call chain.  If you think this method may need to do something with other exceptions (maybe log them?), it will need to handle the generic `Exception`.  The handler could do something with the exception, then re-raise the exception to let other code in the call chain have the chance to handle it, too.